frontend/server/src/Controllers/Problem.php

```diff
@@ -5906,4 +5906,13 @@ public static function getCollectionsDetailsForSmarty(\OmegaUp\Request $r): arra
             'entrypoint' => 'problem_collections_details',
         ];
     }
+
+    /**
+     * @return array{alias: string}
+     */
+    public static function apiRandomProblem(Request $r): string {
```

Comment:
Aquí estás contradiciendo al dockblock, por un lado le dices que el valor de retorno será `array{alias: string}`, y por otro lado le estás indicando que es `string`, así que debes quitar este segundo:
```suggestion
    public static function apiRandomProblem(Request $r) {
```